<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicWebFMRadioPlayWidget</class>
 <widget class="QWidget" name="MusicWebFMRadioPlayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络电台播放窗口</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:white</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>475</width>
     <height>335</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>4</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>网络电台播放窗口</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>457</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>229</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="TTKClickedSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>271</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>274</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>音量</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>475</width>
     <height>310</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255, 255, 255, 100);</string>
   </property>
  </widget>
  <widget class="QLabel" name="lrcLabel">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>34</y>
     <width>270</width>
     <height>260</height>
    </rect>
   </property>
  </widget>
  <widget class="MusicRoundAnimationLabel" name="artistLabel" native="true">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>69</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="previousButton">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>229</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>229</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="positionLabel">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>294</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;font-weight:bold;color:white</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="durationLabel">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>294</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15px;font-weight:bold;color:white;</string>
   </property>
  </widget>
  <widget class="MusicMarqueeWidget" name="titleWidget" native="true">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>34</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="downloadButton">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>305</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>playButton</zorder>
  <zorder>volumeSlider</zorder>
  <zorder>label_2</zorder>
  <zorder>artistLabel</zorder>
  <zorder>previousButton</zorder>
  <zorder>nextButton</zorder>
  <zorder>positionLabel</zorder>
  <zorder>durationLabel</zorder>
  <zorder>titleWidget</zorder>
  <zorder>lrcLabel</zorder>
  <zorder>downloadButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TTKClickedSlider</class>
   <extends>QSlider</extends>
   <header>ttkclickedslider.h</header>
  </customwidget>
  <customwidget>
   <class>MusicMarqueeWidget</class>
   <extends>QWidget</extends>
   <header>musicmarqueewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicRoundAnimationLabel</class>
   <extends>QWidget</extends>
   <header>musicroundanimationlabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
